pages/instructorQuestionEditor/instructorQuestionEditor.ejs

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <%- include('../partials/head'); %>
+    <script type="text/x-mathjax-config">MathJax.Hub.Config({ tex2jax: { inlineMath: [['$','$'], ['\\(','\\)']] }});</script>
+    <script type="text/javascript" src="/MathJax/MathJax.js?config=TeX-MML-AM_CHTML"></script>
+    <script type="text/javascript" src="/javascripts/socket.io.js"></script>
+    <script>
+      document.urlPrefix = '<%= urlPrefix %>';
+      document.csrfToken = '<%= __csrf_token %>';
+    </script>
+    <% if (question.type != 'Freeform') { %>
+    <script type="text/javascript" src="/javascripts/lodash.min.js"></script>
+    <script type="text/javascript" src="/javascripts/require.js"></script>
+    <script type="text/javascript" src="/localscripts/question.js"></script>
+    <script type="text/javascript" src="/localscripts/question<%= effectiveQuestionType %>.js"></script>
+    <% } %>
+
+    <script src="/javascripts/ace/ace.js" type="text/javascript" charset="utf-8"></script>
+    <script src="/javascripts/ace/mode-javascript.js" type="text/javascript" charset="utf-8"></script>
+    <script src="/javascripts/ace/mode-python.js" type="text/javascript" charset="utf-8"></script>
+
+    <script src="/javascripts/PrairieEditor.js"></script>
+  </head>
+  <body>
+    <%- include('../partials/navbar', {navPage: ''}); %>
+    <div id="content" class="container">
+
```

Comment:
No editor for `info.json`?

When `info.json` is saved we will need to call a single-question version of https://github.com/PrairieLearn/PrairieLearn/blob/master/sync/fromDisk/questions.js#L13 (this processes all course questions, so we'll need to factor out a single-question version).